spec/support/signonotron2_integration_helpers.rb

```diff
@@ -35,12 +35,19 @@ def authorize_signonotron_api_user
   end
 
   def load_signonotron_fixture(fixture_sql_file)
-    fixtures_path = Pathname.new(File.join(File.dirname(__FILE__), '../fixtures/integration'))
-    app = "signonotron2"
-    path_to_app = Rails.root.join('..','..','tmp',app)
+    db = YAML.load_file(signon_path + "/config/database.yml")['test']
+    fixture_path = fixtures_path + "/#{fixture_sql_file}"
 
-    db = YAML.load_file(fixtures_path + "#{app}_database.yml")['test']
-    cmd = "sqlite3 #{path_to_app + db['database']} < #{fixtures_path + "#{fixture_sql_file}"}"
+    cmd = "mysql #{db['database']} -u#{db['username']} -p#{db['password']} < #{fixture_path}"
     system cmd or raise "Error loading signonotron fixture"
   end
+
+private
+  def fixtures_path
```

Comment:
@boffbowsh good idea - rebased. I called the method `fixture_file` as I couldn't use `fixture_path` because something active support is dynamically defining a method with that name.